### DISCUSSION TITLE
does anyone know how/what whisper was trained on?

### DISCUSSION THREAD

**Discussion by silvacarl2** [openai/whisper#1492] [Q&A]
does anyone know how/what whisper was trained on?

individual words or entire sentecnes or both?

we are fine tuning a model but not sure what the best approach is.

**Comment by phineas-pta** [comment]
whisper training code is a mystery, openai doesn't want to reveal anything

what stop from u from using sentences for fine tune ?

**Comment by silvacarl2** [comment]
just checking what is the best method for fine tuning whisper.
